### PR TITLE
Pred gear rack and bracer attachments 

### PR DESCRIPTION
**Pull request by Joelampost**
# About the pull request
Removes the claim equipment verb and adds two gear racks that use character preferences for capes and armour.

Now instead of spawning with wrist blades already attached you must get them from the vendor and attach them to your bracer. The code for wrist blades (now bracer attachments) has been refactored also (thank you beagle).

Changes the blooded area of the clan ship to be more open.

Re-sprites all but one of the pred weapons (not done yet).

Gives thwei brain and eye healing properties. 

# Explain why it's good for the game
Spawning items out of thin air as pred has always seemed strange to me so adding a gear rack and having preds share it provides more opportunities for role play between preds.

Being able to mix and match bracer attachments lays the ground work for an unlimited amount of different items and weapons preds can use and having better code for wrist blades (now bracer attachments) cannot be a bad thing.

# Testing Photographs and Procedure
<details>
<summary>Screenshots & Videos</summary>

</details>

# Changelog
:cl: Joe Lampost, Beaglegaming, Sleepynecrons
add: Adds two new vendors for preds
add: Re-sprites all but one of the pred weapons 
balance: Gives thwei brain and eye healing properties.
fix: fixes the on mobs for retro caster
del: Removes being able to choose what accessory you spawn with (adds it to the vendor)
del: Removes the conscious check for big SD laugh so SDing a dead pred triggers the laugh 
del: Removes all of the code related to claim gear
refactor: refactored wrist blade code (now bracer attachments)
soundadd: adds 3 pairs of deployment and retracting sounds for each current bracer attachment
soundadd: adds a sound for attaching attachments to the bracer 
maptweak: Re maps the predator blooded area to be more open 
maptweak: Removes the lightsabre from the pred ship
add: adds two new pred masks
/:cl:

### CODE REVIEW

**Comment by Red-byte3D** [comment]
spaces to tabs

**Comment by cm13-github** [comment]
This pull request has conflicts, please resolve those before we can evaluate the pull request.

File: maps/predship/huntership.dmm
Comment (Nanu308) [inline maps/predship/huntership.dmm:1]: mapping lgtm

**Review by Nanu308 — APPROVED** [review]

**Comment by cm13-github** [comment]
Conflicts have been resolved. A maintainer will review the pull request shortly.

File: code/modules/cm_preds/yaut_bracers.dm
Comment (harryob) [inline code/modules/cm_preds/yaut_bracers.dm:485]:
```suggestion
		if(tgui_alert(user, "Do you want to attach [bracer_attachment] to the left or right hand?", "[src]", list("Right", "Left"), 15 SECONDS) == "Right") //its right, left because in-game itll show up as left, right
```

File: code/modules/cm_preds/yaut_bracers.dm
Comment (harryob) [inline code/modules/cm_preds/yaut_bracers.dm:547]:
```suggestion
	return attachment_internal(usr, FALSE)
```

**Review by harryob — CHANGES_REQUESTED** [review]
little nitpicks

File: code/game/machinery/vending/vendor_types/antag/antag_predator.dm
Comment (realforest2001) [inline code/game/machinery/vending/vendor_types/antag/antag_predator.dm:88]:
```suggestion
/obj/structure/machinery/cm_vending/clothing/yautja
	name = "\improper Yautja Hunting Gear Rack"
	desc = "A gear rack for hunting, expose your bracers ID chip to access."
	icon = 'icons/obj/items/hunter/pred_vendor.dmi'
	icon_state = "pred_vendor_left"
	req_access = list(ACCESS_YAUTJA_SECURE)
	vendor_role = list(JOB_PREDATOR)
	show_points = FALSE
	vendor_theme = VENDOR_THEME_YAUTJA

/obj/structure/machinery/cm_vending/clothing/yautja/can_access_to_vend(mob/user, display = TRUE, ignore_hack = FALSE)
	if(!allowed(user))
		if(display)
			to_chat(user, SPAN_WARNING("Access denied."))
			vend_fail()
		return FALSE

	if(LAZYLEN(vendor_role) && !vendor_role.Find(user.job))
		if(display)
			to_chat(user, SPAN_WARNING("This machine isn't for you."))
			vend_fail()
		return FALSE
	return TRUE

/obj/structure/machinery/cm_vending/clothing/yautja/left_centre
```

File: code/modules/cm_preds/yaut_bracers.dm
Comment (harryob) [inline code/modules/cm_preds/yaut_bracers.dm:510]:
```suggestion
	return remove_attachment_internal(usr, TRUE)
```

**Review by harryob — CHANGES_REQUESTED** [review]

**Comment by harryob** [comment]
otherwise fine, just needs sprite signoff

**Review by harryob — APPROVED** [review]

**Pull request merged**
